### DISCUSSION TITLE
ColumnSeries class does not appear in the namespace OxyPlot.Series

### DISCUSSION THREAD

**Discussion by iSantPy** [oxyplot/oxyplot#2063] [Q&A] [answered]
Hi! I am new in c#, windows forms and OxyPlot. I wanted to make some plot charts and I decided to dowloand OxyPlot to help me with. I did using the NuGet Packages Manager in Visual Studio (2022). Every was ok, but then when I tried to use ColumnSeries to draw a histogram I could not find this class even I was declaring the using OxyPlot.Series at the top of my main file.

The downloaded version of the OxyPlot is 2.1.2. I can't find any solution in the web so I decided to come here and ask.

Thanks in advance!

Ismael

**Comment by VisualMelon** [comment, marked as answer]
`ColumnSeries` was removed at some point: you need to use a `BarSeries` instead, and use the `XAxisKey` and `YAxisKey` to transpose it as necessary.

See https://github.com/oxyplot/oxyplot/discussions/1946 for an example and more discussion.

**Reply by iSantPy** [reply]
Thank you!
